apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -607,28 +609,49 @@ public Response getSymbols(   @Parameter(description = "Cryptocurrency identifie
                                 @Context HttpServletRequest req) throws NotFoundException {
 
         log.debug("getSymbols:  fsym: {}", symbol );
+        TimeZone tz = Calendar.getInstance().getTimeZone();                
         SymbolsOutputDTO symbolsOutputDTO = new SymbolsOutputDTO();
         symbolsOutputDTO.name = symbol;
         symbolsOutputDTO.exchange_traded = "Apollo Currency";
         symbolsOutputDTO.exchange_listed = "Apollo Currency";
-        symbolsOutputDTO.timezone = "America/New_York";
+        symbolsOutputDTO.timezone = tz.getID(); 
         symbolsOutputDTO.minmov = 1;
         symbolsOutputDTO.minmov2 = 0;
         symbolsOutputDTO.pointvalue = 1;
-        symbolsOutputDTO.session = "0930-1630";
-        symbolsOutputDTO.has_intraday = false;
+        symbolsOutputDTO.session = "24x7";
+        symbolsOutputDTO.has_intraday = true;
         symbolsOutputDTO.has_no_volume = false;
+        symbolsOutputDTO.has_daily = true;        
         symbolsOutputDTO.description = "Apollo Foundation.";
-        symbolsOutputDTO.type = "stock";
-        symbolsOutputDTO.supported_resolutions = new ArrayList<>();;
-        symbolsOutputDTO.supported_resolutions.add("D");
-        symbolsOutputDTO.supported_resolutions.add("2D");
-        symbolsOutputDTO.supported_resolutions.add("3D");
+        symbolsOutputDTO.type = "decentralized exchange";
+        symbolsOutputDTO.has_daily = true;
+        symbolsOutputDTO.has_empty_bars = true;
+        symbolsOutputDTO.has_weekly_and_monthly = true;
```

Comment:
our datafeed does not supoort weekly and monthly intervals, so that this field should be set to false.

Reply:
done